src/base-styles/helperClasses/border/radius.css

```diff
@@ -0,0 +1,38 @@
+/*---
+title: Border Radius
+name: borderRadius
+section: Border Helpers
+---
+
+Rounds the corners of an element using a standard
+`border-radius` value.
+
+class            | description
+---------------- | ---------------------------------
+`rounded--top`   | Rounds the top corners of the element
+`rounded--right` | Rounds the right corners of the element
+`rounded--bottom`| Rounds the bottom corners of the element
+`rounded--left`  | Rounds the left corners of the element
+`rounded--all`   | Rounds all corners of the element
+*/
+
+.rounded--top {
+  border-radius: var(--border-radius-default) var(--border-radius-default) 0 0;
+}
+
+.rounded--right {
+  border-radius: 0 var(--border-radius-default) var(--border-radius-default) 0;
+}
+
+.rounded--bottom {
+  border-radius: 0 0 var(--border-radius-default) var(--border-radius-default);
+}
+
+.rounded--left {
+  border-radius: var(--border-radius-default) 0 0 var(--border-radius-default);
+}
```

Comment:
The way you have it, you wouldn't be able to combine `rounded--right` with `rounded--left` because you're zero'ing out the other corners in each declaration. Is this the intention (and if so, should we call this out in documentation)?

Reply:
oh. nvm I see what you're doing :)